libraries/generic/source/generate.js

```diff
@@ -16,20 +16,30 @@ function proxy (value, generator, segments = []) {
   if (typeof value !== 'object' || value === null || Array.isArray(value)) return value
```

Comment:
why so much honor to the array? according to the idea, it should also work like for example Set.

Reply:
39676d1aa36dd43e9319c9a7410149e7365403fe